### DISCUSSION TITLE
SB350 ok but error on Solo

### DISCUSSION THREAD

**Discussion by JonG-S** [GerryDazoo/Slinger#267] [Q&A]
Delighted with results on 350 but server shows this error when trying to connect to Solo
Slinginfo  "Solo/Pro/ProHD" 5 30 192.168.9.254 5001 3000 10
Solo Starting Stream for  192.168.9.84:10790
Connecting... ('192.168.9.254', 5001) Control
cmd: 0x67 err: 0x2 8
Solo Error Starting Session. Check your admin password in config.ini file!
Solo ERROR: Slingbox session startup failed.
Solo Logging Out
Exception in thread Thread-3 (streamer):
Traceback (most recent call last):
  File "threading.py", line 1016, in _bootstrap_inner
  File "threading.py", line 953, in run
  File "slingbox_server.py", line 964, in streamer
    if s_ctl : s_ctl = closecontrol(s_ctl)
  File "slingbox_server.py", line 576, in closecontrol
    if Solo: sling_cmd( 0x68, b'')
  File "slingbox_server.py", line 380, in sling_cmd
    sid, stat, dlen = unpack("2x H 8x H 2x H", response[0:18] ) # "x2 v x8 v x2 v", $hbuf);
struct.error: unpack requires a buffer of 18 bytes

Any assistance much appreciated, thanks

**Comment by reg7** [comment]
> Delighted with results on 350 but server shows this error when trying to connect to Solo
> 
> Slinginfo  "Solo/Pro/ProHD" 5 30 192.168.9.254 5001 3000 10
> 
> Solo Starting Stream for  192.168.9.84:10790
> 
> Connecting... ('192.168.9.254', 5001) Control
> 
> cmd: 0x67 err: 0x2 8
> 
> Solo Error Starting Session. Check your admin password in config.ini file!
> 
> Solo ERROR: Slingbox session startup failed.
> 
> Solo Logging Out
> 
> Exception in thread Thread-3 (streamer):
> 
> Traceback (most recent call last):
> 
>   File "threading.py", line 1016, in _bootstrap_inner
> 
>   File "threading.py", line 953, in run
> 
>   File "slingbox_server.py", line 964, in streamer
> 
>     if s_ctl : s_ctl = closecontrol(s_ctl)
> 
>   File "slingbox_server.py", line 576, in closecontrol
> 
>     if Solo: sling_cmd( 0x68, b'')
> 
>   File "slingbox_server.py", line 380, in sling_cmd
> 
>     sid, stat, dlen = unpack("2x H 8x H 2x H", response[0:18] ) # "x2 v x8 v x2 v", $hbuf);
> 
> struct.error: unpack requires a buffer of 18 bytes
> 
> 
> 
> 
> 
> Any assistance much appreciated, thanks

This has been covered extensively in other threads. It checks the password and the session then fails to start, pointing to an authentication problem.

**Reply by reg7** [reply]
... or at least that was my issue. I was seeing the same error in my logs.

**Reply by JonG-S** [reply]
Thank you - my apologies for not having seen this elsewhere.  I will search "better"!

However I checked the password both from "https://newwatchsecure.slingbox.com/watch/slingAccounts/account_boxes_js" and from the logs on my machine so bit lost.

**Reply by reg7** [reply]
Well then you did do your homework. In this case, I should let the more knowledgeable contributors comment :)

**Comment by GerryDazoo** [comment]
Your Solo had been bricked by Mr. Sling.
